app/src/main/java/com/calendar/MainActivity.java

```diff
@@ -42,65 +40,21 @@ public class MainActivity extends AppCompatActivity {
     private final ArrayList<Event> eventList = new ArrayList<>();
     private Adapter adapter;
     private RecyclerView recyclerView;
-    private CalendarView calendarView;
-    private DatePickerDialog.OnDateSetListener dateSetListener;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        calendarView = findViewById(R.id.calendarView);
+        FragmentManager fragmentManager = getSupportFragmentManager();
+        fragmentManager.beginTransaction()
+                .replace(R.id.calendarFragmentContainer, MonthlyViewFragment.class, null)
+                .setReorderingAllowed(true)
+                .addToBackStack("name") // name can be null
+                .commit();
+
         databaseReference = FirebaseDatabase.getInstance().getReference("Calendar");
         setEventRecyclerView();
-        notifyChange();
-
-        calendarView.setOnDateChangeListener((view, year, month, dayOfMonth) -> {
-            CalendarUtils.selectedDate = LocalDate.of(year, month + 1, dayOfMonth);
-            notifyChange();
-        });
-    }
-
-    @Override
-    protected void onResume() {
```

Comment:
Are you sure that we don't need to handle on onResume ?

Reply:
It spoils saving date state during minimize and maximize application.